versions/build.hercules.ver

```diff
@@ -1,3 +1,5 @@
+export spack_mod_path="/work/noaa/epic/role-epic/spack-stack/hercules/spack-stack-1.6.0/envs/gsi-addon-env/install/modulefiles/Core"
```

Comment:
Could these `spack_mod_path` variables use `spack_stack_ver` and `spack_env` variables so we can reduce the places we'd have to update those when they change?

Reply:
Thanks for making the change! Looks good!